tests/TestCase/Utility/InflectorTest.php

```diff
@@ -204,80 +218,94 @@ public function testSingularizeMultiWordIrregular()
     /**
      * testInflectingPlurals method
      *
+     * @dataProvider pluralizeProvider
      * @return void
      */
-    public function testInflectingPlurals()
+    public function testInflectingPlurals($plural, $singular)
+    {
+        $this->assertEquals($plural, Inflector::pluralize($singular));
```

Comment:
This is what output looks like on a failure, so pretty easy to see what's causing failure:

```
1) Cake\Test\TestCase\Utility\InflectorTest::testInflectingPlurals with data set #52 ('potatoes', 'potatox')
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'potatoes'
+'potatoxes'
```